site/en/tutorials/audio/music_generation.ipynb

```diff
@@ -680,7 +680,7 @@
         "id": "xIBLvj-cODWS"
       },
       "source": [
-        "Next, create a [tf.data.Dataset](https://www.tensorflow.org/datasets) from the parsed notes."
+        "Next, create a [tf.data.Dataset](https://www.tensorflow.org/api_docs/python/tf/data/Dataset) from the parsed notes."
```

Comment:
Good catch. We could also use the full API name and the link should get auto-generated.

```suggestion
        "Next, create a `tf.data.Dataset` from the parsed notes."
```